src/translations/translations.en.js

```diff
@@ -1642,6 +1642,36 @@ export const translations = {
   'cc-token-api-list.revoke-token': /** @param {{ name: string}} _ */ ({ name }) => `Revoke API token - ${name}`,
   'cc-token-api-list.revoke-token.error': `Something went wrong while revoking the API token`,
   'cc-token-api-list.revoke-token.success': `The API token has been revoked successfully`,
+  'cc-token-api-list.update-token': `Modify`,
+  'cc-token-api-list.update-token-with-name': /** @param {{ name: string}} _ */ ({ name }) =>
+    `Modify API token - ${name}`,
+  //#endregion
+  //#region cc-token-api-update-form
+  'cc-token-api-update-form.back-to-list': `Back to the API token list`,
+  'cc-token-api-update-form.cli.content': () => sanitize`
+    <p>
+      Manage your API tokens from a terminal using the commands below.
+      To install Clever Tools CLI, follow the instructions from the <a href="https://www.clever-cloud.com/developers/doc/cli/install/" title="documentation - Install Clever Tools - new window">documentation</a>.
+    </p>
+    <dl>
+      <dt>Create a token:</dt>
+      <dd><code>clever tokens create "&lt;your token name&gt;"</code></dd>
+      <dt>Revoke a token:</dt>
+      <dd><code>clever tokens revoke &lt;api_token_id&gt;</code></dd>
+      <dt>List tokens:</dt>
+      <dd><code>clever tokens list</code></dd>
+      <dt>Use your API token&nbsp;:</dt>
+      <dd><code>curl -H "Authorization: Bearer &lt;your_token&gt;" https://api-bridge.clever-cloud.com/v2/self</code></dd>
```

Comment:
thought: having "your_token" here and "api_token_id" above is confusing, is it the "api_token_id" that you're supposed to use here? If so, could we use the same wording?